CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
-# 2.4.1 [#73](https://github.com/patterninc/muffin_man/pull/73)
+# 2.4.1
```

Comment:
Gavin has updated the version once or twice now, so this should probably be moved to the current version